ansible/roles/dev-desktop/files/scripts/init_git.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+import requests
+import re
+import subprocess
+import getpass
+
+for cfg in ["email", "name"]:
+    if subprocess.run(["git", "config", "--global", f"user.{cfg}"], stdout=subprocess.DEVNULL).returncode == 0:
+        print("git username and email already configured")
+        exit(0)
+
+dev_desktop = requests.get("https://team-api.infra.rust-lang.org/v1/permissions/dev_desktop.json")
+dev_desktop.raise_for_status()
+
+people = requests.get("https://team-api.infra.rust-lang.org/v1/people.json")
+people.raise_for_status()
+people_map = people.json()["people"]
+
+username = getpass.getuser()
+username = username.lstrip("gh-")
+
+print(f"found github name: {username}")
+
+for github_name in dev_desktop.json()["github_users"]:
+    if github_name == username:
+        if github_name not in people_map:
+            print("could not find a matching user in the people map!")
+            continue
```

Comment:
I might be missing something, but why are we checking `dev_desktop.json` first? We already know that the `username` is the GitHub name, based on the conventions that are used to create the user accounts. And the user cannot run the script without being in that file. Instead of fetching `dev_desktop.json` and checking there first, can't we simply look for the user in `people.json`?

```python
if username not in people_map:
    print("could not find a matching user in the people map!")
    continue
```